ChangeLog.md

```diff
@@ -10,6 +10,8 @@
 
 - Defines a new `Has` constraint synonym, conveniently combining `Carrier` and `Member` constraints and used for all effect constructors. ([#217](https://github.com/fused-effects/fused-effects/pull/217))
 
+- Allows effects to be defined and handled as sums of other effects, while still using the constructors for the component effects. This has been used to redefine `NonDet` as a sum of `Empty` and `Choose`. ([#199](https://github.com/fused-effects/fused-effects/pull/199), [#219](https://github.com/fused-effects/fused-effects/pull/219))
```

Comment:
Can we note in here an example of how effect composition used to work, and how people should port their existing code to the new style?

Reply:
This isn’t really a change to effect composition, and doesn’t require any porting at all. It just gives you an extra tool to e.g. split effect constructors up into separate effects when they have some independent value.

Reply:
I was extremely confused. My apologies!